timesketch/api/v1/resources/__init__.py

```diff
@@ -232,7 +232,12 @@ def datastore(self):
         """
         return ElasticsearchDataStore(
             host=current_app.config['ELASTIC_HOST'],
-            port=current_app.config['ELASTIC_PORT'])
+            port=current_app.config['ELASTIC_PORT'],
```

Comment:
these are a lot of options that you need to read in, and configure and in a lot of places.

I think it would be much better to do this in a shared library, maybe just in the `timesketch.lib.datastores.elastic` or in some other file, maybe a new one, that can the import both this library as well as the flask configs, that would set it up, ... since you both have a lot of options you are reading and having conditions for SSL oro fallback to HTTP

Reply:
Pulled the SSL config into timesketch/lib/datastores/elastic.py.

Looking at timesketch/lib/aggregators/interface.py we skip the datastore and use the Elasticsearch library directly.

Is there a reason behind this, or would we be okay to implement the methods used under ElasticsearchDataStore?